Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN apt-get update && \
     apt-get install -y --no-install-recommends \
     sudo=1.8.* \
     lsb-release=11.1.* \
-    software-properties-common=0.98.* \
+    software-properties-common \
```

Comment:
@davidkarlsen @Otterian Was there a reson for pinning this to `0.98.*`? It has a static dependency - pinning that `python3-software-properties` to the same version also works.